pkg/kubeinteraction/labels.go

```diff
@@ -85,4 +86,10 @@ func AddLabelsAndAnnotations(event *info.Event, pipelineRun *tektonv1.PipelineRu
 	for k, v := range annotations {
 		pipelineRun.Annotations[k] = v
 	}
+
+	// Add annotations to PipelineRuns to integrate with Tekton Results
+	err := AddResultsAnnotation(event, pipelineRun)
+	if err != nil {
+		logger.Errorf("Failed to add results annotations to PipelineRuns with error: %w", err)
```

Comment:
I think we need to handle the error here

Earlier there was no error to return from `AddLabelsAndAnnotations` so there was no return but now `AddResultsAnnotation` retruns error and if you don't handle we miss the failure

Reply:
we want to return the error and return it back, to the initial callers, since we have some logic to take that error and echo it in the user repo namespace